src/PhpBrew/PatchKit/Patch.php

```diff
@@ -24,15 +24,15 @@ abstract class Patch
      *
      * @return array<PatchRule>
      */
-    abstract public function rules();
+    abstract public function rules(Buildable $build, Logger $logger);
```

Comment:
Please revert and do as suggested in https://github.com/phpbrew/phpbrew/issues/1040#issuecomment-562217403.

Reply:
Done.